Libraries/Opc.Ua.Security.Certificates/System.Security.Cryptography.X509Certificates/RSAPkcs1X509SignatureGenerator.cs

```diff
@@ -0,0 +1,107 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NETSTANDARD2_1
+
+// This source code is intentionally copied from the .NET core runtime to close
+// a gap in the .NET 4.6 and the .NET Core 2.x runtime implementations.
+// original code is located here:
+// https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/RSAPkcs1X509SignatureGenerator.cs
+#pragma warning disable CS1591 // Suppress missing XML comments to preserve original code
+
+using System;
+using System.Diagnostics;
+using System.Formats.Asn1;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Opc.Ua.Security.Certificates
+{
+    internal sealed class RSAPkcs1X509SignatureGenerator : X509SignatureGenerator
```

Comment:
Why do you need this class?  What's it doing differently than the built-in RSA one?  `X509SignatureGenerator.CreateForRSA(key, RSASignaturePadding.Pkcs1)`

Reply:
Nevermind, I see that this is .NET Standard 2.0 compatibility code.